packages/enzyme/src/selectors.js

```diff
@@ -362,10 +363,22 @@ function matchDescendant(nodes, predicate) {
  * the selector. The selector can be a simple selector, which
  * is handled by `buildPredicate`, or a complex CSS selector which
  * reduceTreeBySelector parses and reduces the tree based on the combinators.
- * @param {Function|Object|String} selector
+ *
+ * @param {EnzymeSelector} selector
  * @param {RSTNode} root
  */
 export function reduceTreeBySelector(selector, root) {
+  if (typeof selector !== 'string') {
+    const elements = elementsByConstructor(selector);
```

Comment:
This pops "Update your Java installation" in IE11 😢 
https://github.com/ljharb/html-element-map/issues/3